README.md

```diff
@@ -2,34 +2,38 @@
 
 # Bluetooth, Airplay and Spotify audio streaming for any audio device
 
-**Starter project enabling you to add audio streaming via Bluetooth, Airplay or Spotify Connect to any old speakers or Hi-Fi using just a Raspberry Pi.**
+**Starter project enabling you to add multi-room audio streaming via Bluetooth, Airplay or Spotify Connect to any old speakers or Hi-Fi using just a Raspberry Pi.**
 
-This project has been tested on Raspberry Pi 3B/3B+ and Raspberry Pi Zero W. If you're using a Raspberry Pi 3 or above you don't need any additional hardware but if you'd like to use a Pi Zero W this will require an additional HAT as this model has no audio output.
+**Features**
+- **Bluetooth, Airplay and Spotify Connect support**: Stream audio from your favourite music services or directly from your smartphone/computer using bluetooth.
+- **Multi-room synchronous playing**: Play perfectly synchronized audio on multiple devices all over your place.
 
-## Hardware required
+### Hardware required
 
-* Raspberry Pi 3A+/3B/3B+/Zero W
+* Raspberry Pi 3A+/3B/3B+/4B/Zero W
```

Comment:
We should probably recommend multi room is disabled for the Pi Zero - it kinda sorta works but breaks up a lot at the start of each track

Reply:
I asume this is with the Pi Zero as the master server, correct?
If this is the case, can you verify if using it as a client works fine? 

I think it's a good idea for me to get a DAC, any recommendations?

Reply:
@tmigone yeah, I am looking at the scenario where a single device user doesn't care about multi-room, we don't want to break their setup due to them updating. I tested a Pi Zero on it's own, so it *was* operating as the master even though there were no clients, and it wasn't usable.

As for DACs, it would be great to support the new Pimoroni ones, since they discontinued the PHAT DAC. Take your pick: https://shop.pimoroni.com/collections/pirate-audio

Reply:
Thanks. 

Regarding the Pi Zero, did you try using the `DISABLE_MULTI_ROOM` var? Using it should result in the project running without `snapcast`, so same as before this update. This is actually the reason behind having this update as a major change, it breaks or changes the functionality fundamentally unless you apply that env var.
In any case, I'll test this again tomorrow with my Pi Zero to have another data point. Will hold the merge until we get a better picture.

Reply:
@tmigone yes I did - that's why I initially recommended that we say that, but definitely go ahead and test as well to get that second data point.

Reply:
@chrisys to clarify, when you used the flag (`DISABLE_MULTI_ROOM`) it worked fine, correct? 
Can you test with the Pi Zero as a client?